Gemfile

```diff
@@ -41,7 +41,8 @@ gem "tzinfo-data", platforms: %i[ mingw mswin x64_mingw jruby ]
 
 # Reduces boot times through caching; required in config/boot.rb
 gem "bootsnap", require: false
-
+gem 'devise', '~> 4.8', '>= 4.8.1'
+gem 'devise_token_auth', '>= 1.2.0', git: "https://github.com/lynndylanhurley/devise_token_auth"
```

Comment:
Are you explicity setting the git repo for a particular reason? if not, maybe you can remove the git reference so it will be fetched from rubygems.org as all other gems